app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -387,45 +376,97 @@ class BrowserTabViewModel(
         browserViewState.value = currentState.copy(isFullScreen = false)
     }
 
-    override fun loadingStarted(url: String?) {
-        Timber.v("Loading started")
-        if (!currentBrowserViewState().browserShowing) return
-        val progress = currentLoadingViewState()
-        loadingViewState.value = progress.copy(isLoading = true)
-        pendingUrl = url
-        site = null
-        onSiteChanged()
-    }
+    override fun navigationStateChanged(newWebNavigationState: WebNavigationState) {
```

Comment:
lint warning that param name doesn't match the supertype and may cause problems

Reply:
Oh, I really need to turn this lint option on!